taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -58,13 +65,23 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         setOpen(false);
     }
 
+    // Used for toggling visibility of a translation language
+    const handleToggleTranslationVisibility = (key) => {
+        const newShownTranslationLanguages = shownTranslationLanguages.includes(key) ? shownTranslationLanguages.filter(obj => !(key === obj)) : [...shownTranslationLanguages, key]
+        setShownTranslationLanguages(newShownTranslationLanguages);
+        newShownTranslationLanguages.sort()
+        localStorage.setItem('shownTranslationLanguages', JSON.stringify(newShownTranslationLanguages))
```

Comment:
nit: maybe have this key name (`shownTranslationLanguages`) as a const outside of the component, just so is easier to keep track of it